targets/freertos.mips4k.pic32mx/freertos_drivers/plib_pic32mx530/sources

```diff
@@ -0,0 +1,36 @@
+DEPS += PIC32MXLIBPATH
+
+PLIBS := can int osc pcache
+
+PLIBDIRS := $(foreach PNAME,$(PLIBS),$(PIC32MXLIBPATH)/pic32-libs/peripheral/$(PNAME)/source )
+
+#CSRCS=$(foreach file,$(foreach dir,$(PLIBDIRS),$(wildcard $(dir)/*.c)),$(notdir $(file)))
+#CXXSRCS=$(foreach file,$(foreach dir,$(PLIBDIRS),$(wildcard $(dir)/*.cxx)),$(notdir $(file)))
+#CPPSRCS=$(foreach file,$(foreach dir,$(PLIBDIRS),$(wildcard $(dir)/*.cpp)),$(notdir $(file)))
+
+define PDIR_INCLUDE_template
+include $(PIC32MXLIBPATH)/pic32-libs/peripheral/$(1)/source/Makefile.inc
+endef
+
+$(foreach PNAME,$(PLIBS),$(eval $(call PDIR_INCLUDE_template,$(PNAME))))
+
+#include $(PIC32MXLIBPATH)/pic32-libs/peripheral/can/source/Makefile.inc
+#include $(PIC32MXLIBPATH)/pic32-libs/peripheral/int/source/Makefile.inc
+#include $(PIC32MXLIBPATH)/pic32-libs/peripheral/osc/source/Makefile.inc
+
+CORECFLAGS += -D_SUPPRESS_PLIB_WARNING
```

Comment:
let's add the -D__PIC32MX530....__ compiler flag here, and the sources file inside all other processor-specific directory (e.g. the drivers_ for both 795 and 530 as well as plib for 795 and 530 both).